packages/framework/fluid-static/api-report/fluid-static.api.md

```diff
@@ -44,7 +44,7 @@ export function createServiceAudience<TMember extends IMember = IMember>(props:
 // @public
 export type DataObjectClass<T extends IFluidLoadable = IFluidLoadable> = {
     readonly factory: {
-        IFluidDataStoreFactory: DataObjectClass<T>["factory"];
+        readonly IFluidDataStoreFactory: DataObjectClass<T>["factory"];
```

Comment:
This change seems good on its own. Should we split this into a separate PR?

Reply:
DOne.